readme.txt

```diff
@@ -1 +1,21 @@
-Ninslash.com
\ No newline at end of file
+Ninslash.com
+
+# Compilation (Ubuntu) #
+
+## Needed libraries ##
+
+```
+sudo apt-get install python alsa (asound) gl glu x11 libsdl freetype bam
+```
+
+## Compilation ##
+
+Download the sources using git:
+
+```
+git clone https://github.com/Siile/Ninslash.git
+cd ninslash
+bam release
```

Comment:
I would rather recommend to use `bam client_release` to build the client and `bam server_release` to build the server. This only builds what the user wants, and not all this useless tools stuff nobody wants.
If you want to build both at once, we can put em into one command and additionally enable multiprocessing to get the fastest build time:
```
bam -j $(nproc) client_release server_release
```